src/applications/mhv/landing-page/components/tests/LandingPage.test.unit.spec.jsx

```diff
@@ -1,14 +0,0 @@
-import React from 'react';
```

Comment:
NITPICK: Should probably update this instead of removing it, but since it's not doing much, this can be done in a later PR.